lib/auth/vault.js

```diff
@@ -80,7 +80,10 @@ function vaultSignatureCb(err, authInfo, log, callback, streamingV4Params) {
  */
 function authenticateV2Request(params, requestContexts, callback) {
     params.log.debug('authenticating V2 request');
-    const serializedRCsArr = requestContexts.map(rc => rc.serialize());
+    let serializedRCsArr;
+    if (requestContexts) {
```

Comment:
From what I understood, we always at least construct an array, even if empty. Am I mistaken?